### ISSUE TITLE
bootprint-openapi has peerDependency to old bootprint

### ISSUE THREAD

**Issue by creinig** [bootprint/bootprint-monorepo#5]
When installing bootprint-openapi via `npm install -g bootprint bootprint-openapi` I get the following error:

```
npm WARN bootprint-openapi@4.0.2 requires a peer of bootprint@^2.0.1 but none was installed.
```

Installing with `npm install -g bootprint@2.0.1 bootprint-openapi` in turn results in

```
npm WARN bootprint-json-schema@4.0.2 requires a peer of bootprint@^4.0.1 but none was installed.
npm WARN bootprint-base@4.0.2 requires a peer of bootprint@^4.0.1 but none was installed.
```

I *assume* the peerDependency from `bootprint-openapi@4.0.2` to `bootprint@2.0.1` is a typo and should instead read `bootprint@4.0.1` (since it has been updated just 10 days ago)?

**Comment by nknapp** [comment]
That's right. It should be 4.0.2

Actually, if it had a devDependency to bootprint, the peerDependency would be updated as well. I'll fi that

**Comment by nknapp** [comment]
should be solved with 4.0.3
